youtube_dl/extractor/svt.py

```diff
@@ -360,11 +364,26 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, playlist_id)
 
+        redux_state = self._parse_json(
+            self._search_regex(
+                r'\.\s*reduxState=\s*({.*})',
```

Comment:
Do not capture empty dict.

Reply:
Would a simple `{.+}` suffice as replacement?